### DISCUSSION TITLE
Legal Last Name

### DISCUSSION THREAD

**Discussion by shubhamsh5447** [community/community#56331] [GitHub Education]
### Hi everyone,

I don't have legal last name, So it is being rejected everytime. Please guide me what to do for student developer pack.

**Comment by samus-aran** [comment]
Hi @shubhamsh5447 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 I suspect this would help you in your process.
